src/skin/skinloader.cpp

```diff
@@ -56,12 +56,19 @@ QString SkinLoader::getSkinPath(const QString& skinName) const {
     return QString();
 }
 
-QPixmap SkinLoader::getSkinPreview(const QString& skinName) const {
-    QPixmap preview(getSkinPath(skinName) + "/preferences_preview_screenshot.png");
+QPixmap SkinLoader::getSkinPreview(const QString& skinName, const QString& schemeName) const {
+	QPixmap preview(":/images/skin_preview_placeholder.png");
```

Comment:
I wonder if it's better to initialize `preview` empty and load the default image only when necessary, instead of loading the default and then overwriting it. Probably doesn't make a difference performance-wise but it seems smarter.
?